OpenOversight/migrations/versions/0ed957db0058_add_description_to_officers.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
```

Comment:
It's not mandatory, but always helpful to add a description here. It should auto-populate if you create the migration with `python manage.py db migrate -m 'Add column foo'`.